cost/azure/rightsize_managed_disks/README.md

```diff
@@ -0,0 +1,61 @@
+# Azure Rightsize Managed Disks
+
+## What it does
+
+This Policy Template scans all volumes in the given account and identifies any volume that meets the user-specified criteria for being oversized. The user can filter volumes based on usage percentage of capacity (GiB used), usage percentage of IOPS, usage percentage of throughput, or any combination of these. Any volumes that meet the user-specified criteria are considered oversized. If any oversized volumes are found, an incident report will show the volumes and related information. An email will be sent to the user-specified email addresses.
+
+- It is important that you keep constant LUN numbers for your data disks because when policy retrieves metrics for them, it will only look for the latest LUN number assigned to the data disk, if this one is changed multiple times before running the policy, only the data points using the current LUN the disk has will be retrieved for analysis.
```

Comment:
There are multiple sentences here separated by commas instead of periods or semicolons and a grammatical error. This is what it should look like:

It is important that you keep constant LUN numbers for your data disks because when policy retrieves metrics for them. It will only look for the latest LUN number assigned to the data disk; if changed multiple times before running the policy, only the data points using the current LUN the disk has will be retrieved for analysis.

Reply:
Thank you @XOmniverse , I applied these changes!